dae/package.nix

```diff
@@ -4,7 +4,7 @@
 }:
 buildGoModule rec {
   pname = "dae";
-  version = "unstable-2023-09-04";
+  version = "unstable-2024-02-14";
```

Comment:
According to <https://github.com/NixOS/nixpkgs/blob/e2a0450b4b32da75fb3166c86087d79963020bff/pkgs/README.md?plain=1#L353>, version composed with the date of commit.